recipe/meta.yaml

```diff
@@ -54,3 +54,6 @@ about:
 extra:
   recipe-maintainers:
     - sodre
+    - oblute
+    - rluria14
+    - ndmaxar
```

Comment:
We should wait on the team maybe?

Also if there are going to be a bunch of these, it might make sense to automate this. I can help with that.

Reply:
@beckermr myself, rluria14, and ndmaxar are working on this together, so will it be necessary to require approval at every PR?

Reply:
would love any help with automating this. we have a list of the 42 that are choked up with many PRs and outdated

Reply:
yup! see the issue I just opened and bumped you all about in gitter. If we get agreement there, I can automate the process and it will all get done!